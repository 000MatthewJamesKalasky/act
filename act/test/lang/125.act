defproc bar(bool a, b)
{
  prs {
    a => b-
  }
}

deftype myfoo <: bool (bool t, f) { }

defproc test()
{
  bool x, y;
  prs {
   
    x => y-
  }

  [ true ->

  refine +{ myfoo x; }
  {
    bar q(x.t,y);
  }
  ]
}

test t;
