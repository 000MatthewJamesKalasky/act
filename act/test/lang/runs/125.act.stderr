ERROR: File `125.act', line 20, col 19
	Refinement overrides cannot be within a conditional construct!
